.gitignore

```diff
@@ -12,3 +12,7 @@ yarn.lock
 # IDEs
 .idea/
 .vs/
+
+config.json
+config.*.json
+!config.example.json
```

Comment:
Why is this excluded?

Reply:
`config.json` and `config.*.json` exclude customized configurations.

`config.example.json` is the auto generated configuration example and explizitly included because it matches `config.*.json`.